### DISCUSSION TITLE
PySide2 app Android package

### DISCUSSION THREAD

**Discussion by gusarg81** [beeware/briefcase#593] [Q&A]
Hi,

Before all, I am new to briefcase (Beeware as well). I found briefcase after starting to play with Beeware. Maybe is an stupid question, but: If I create the package for android of an pyside2 project, would it work on Android?

That being said, I've successfully created the .apk package (briefcase package android) of the default app (briefcase new and choosing PySide2 template). I've installed in Android, run it but after Beeware Splash it crashes.

Thanks.

**Comment by freakboy3742** [comment]
In theory, it *could* work, but it's not an officially supported configuration at present. For the moment, Toga is the only mobile platform that Briefcase supports.

We'd definitely be interested in patches to address this. In the meantime, we need to improve documentation to clarify the situation.

**Comment by gusarg81** [comment]
@freakboy3742 OK! Thanks!
